doc/examples/simulate_multi_tensor.py

```diff
@@ -76,7 +76,7 @@
 """
 For the ODF simulation we will need a sphere. Because we are interested in a
 simulation of only a single voxel, we can use a sphere with very high
-resolution. We generate that by subdividing the triangles of one of Dipy's
+resolution. We generate that by subdividing the triangles of one of dipy_'s
```

Comment:
I guess it's because the `link_names.inc` file was NOT included. Fixed in 412e9fb.